packages/apps/human-app/frontend/src/modules/worker/oracle-registration/registration-form.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Stack } from '@mui/material';
+import { FormProvider } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { Button } from '@/shared/components/ui/button';
+import { HCaptchaForm } from '@/shared/components/hcaptcha/h-captcha-form';
+import { useOracleRegistrationFlow, useRegistrationForm } from './hooks';
+
+export function RegistrationForm() {
+  const { t } = useTranslation();
+  const methods = useRegistrationForm();
+  const {
+    hasViewedInstructions,
+    handleInstructionsView: onInstructionsClick,
```

Comment:
Generally, naming convention in react is:
- `onEventName` - for prop names of component, that access some event handler
- `handleEventName` - for handlers of some events that you pass to `onEventName` prop

So it's fine to keep `handleInstructionsView` (and others) name as is and simply use it below